rye/src/pyproject.rs

```diff
@@ -349,8 +349,7 @@ impl Workspace {
 
     /// Checks if a project is a member of the declared workspace.
     pub fn is_member(&self, path: &Path) -> bool {
-        let canonicalized = self.root.join(path);
-        if let Ok(relative) = path.strip_prefix(canonicalized) {
+        if let Ok(relative) = path.strip_prefix(&self.root) {
```

Comment:
```suggestion
        let absolutized = self.root.join(path);
        if let Ok(relative) = absolutized.strip_prefix(&self.root) {
```

It's possible this is how it was intended. I don't off hand know which code paths it would make a difference (do we get relative paths as input from somewhere?)

Reply:
I'm not sure if you can get relative inputs. In theory should not happen.